src/gui/sdlmain.cpp

```diff
@@ -3008,11 +3027,13 @@ void Config_Add_SDL() {
 	Pmulti->Set_help("Priority levels for dosbox. Second entry behind the comma is for when dosbox is not focused/minimized.\n"
```

Comment:
Last thing I can think of is using auto-by-default (just above this line here)
`Pmulti->SetValue("higher,normal");` -> `Pmulti->SetValue("auto, auto");`

Reply:
you sure you want to jump in with that as the default right away? 😀

Reply:
Like the PR that moved pixel format decisions from DOSBox's realm back into SDLs (which should know best), this PR also moves application scheduling decisions from DOSBox's realm back into the OS scheduler - which also should also know best.

I think that's a safe proposition given the evolution of modern OS schedulers; of course - we'll test first.

(I also don't want to disparage this original code - certainly schedulers back in the Windows 2000 and XP and Linux 2.4 kernel days /did/ need some strong-arming to get steady low-latency performance for games).

Reply:
Testing begins, thanks @kklobe !
We'll have broader testing when we generate the pre-release 0.77 binary too.
We'll need a good reason through, give this PR already improves scheduling on macOS :-)